src/actions/NotificationActions.js

```diff
@@ -243,8 +249,8 @@ export const newPriceChangeEvent = (
         )}`
       ],
       pluginId: currencyInfo.pluginId,
-      dailyChange: dailyChangeEnabled ? 10 : undefined,
-      hourlyChange: hourlyChangeEnabled ? 3 : undefined
+      dailyChange: dailyChangeEnabled ? 0.0001 : undefined,
+      hourlyChange: hourlyChangeEnabled ? 0.0003 : undefined
```

Comment:
What is up with these numbers? For testing? They seem too small.

Reply:
Yup for testing and removed in the force pushed branch